java-frontend/src/main/java/org/sonar/java/bytecode/se/BytecodeEGWalker.java

```diff
@@ -555,12 +705,21 @@ private boolean handleMethodInvocation(Instruction instruction) {
             }));
         methodInvokedBehavior
           .exceptionalPathYields()
-          .forEach(yield ->
+          .forEach(yield -> {
+            org.sonar.plugins.java.api.semantic.Type exceptionType = yield.exceptionType();
             yield.statesAfterInvocation(
-              Lists.reverse(pop.values), Collections.emptyList(), pop.state, () -> constraintManager.createExceptionalSymbolicValue(yield.exceptionType())).forEach(ps -> {
-              ps.storeExitValue();
-              enqueue(new ProgramPoint(exitBlock), ps);
-            }));
+              Lists.reverse(pop.values), Collections.emptyList(), pop.state, () -> constraintManager.createExceptionalSymbolicValue(exceptionType)
+              ).forEach(ps -> {
+                ps.storeExitValue();
```

Comment:
I don't see a test where this is used (should be an invocation throwing an exception which is not caught )

Reply:
assertion added.